core/common/src/test/java/alluxio/AlluxioTestDirectory.java

```diff
@@ -60,7 +57,6 @@ public static File createTemporaryDirectory(File parent, String prefix) {
       throw new RuntimeException("Failed to create directory " + file.getAbsolutePath());
     }
 
-    Runtime.getRuntime().addShutdownHook(new Thread(() -> delete(file)));
```

Comment:
conventionally i'm guessing this should only be used when `File parent` is a subdirectory within a testing directory created by `createTestingDirectory()`, but nothing actually guarantees that. is there an issue with keeping this shutdown hook besides accumulating a lot of them? does this accumulation affect the stability of tests in any way?

Reply:
I would prefer all temp dir to be under the same subdir for the entire testing process, so changing this to a private method.